cmd/eks-node-viewer/main.go

```diff
@@ -29,19 +44,30 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/homedir"
 
 	"github.com/awslabs/eks-node-viewer/pkg/client"
 	"github.com/awslabs/eks-node-viewer/pkg/model"
 	"github.com/awslabs/eks-node-viewer/pkg/pricing"
 )
 
+var kubeconfig *string
+
 func main() {
 	nodeSelectorFlag := flag.String("nodeSelector", "", "Node label selector used to filter nodes, if empty all nodes are selected ")
 	resources := flag.String("resources", "cpu", "List of comma separated resources to monitor")
 	disablePricing := flag.Bool("disable-pricing", false, "Disable pricing lookups")
-
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
 	flag.Parse()
-	cs, err := client.Create()
+	kubeconfigenv := os.Getenv("KUBECONFIG")
```

Comment:
If this is checked after the flags are parsed then KUBECONFIG environment variable overrides someone explicitly setting `-kubeconfig` at the CLI.

I didn't realized there was a PR for this already and changed the default value if KUBECONFIG was set in #10